content/apidocs-mxsdk/apidocs/team-server-api.md

```diff
@@ -6,201 +6,444 @@ menu_order: 65
+
+#### 4.4.3 Response Payload
 
 List of objects with the following key-value pairs:
 
-*   _Number_ (Long) : Number of the revision.
-*   _CommitMessage_ (String) : Commit message of the revision.
-*   _Date_ (Date) : Date when the revision is created (or the commit is done).
-*   _Author_ (String) : Creator of the revision (committer).
-*   _MendixVersion_ (String) : Version string of the Mendix version of the app in this revision.
+|Name|Type|Description
+|---|---|---|
+|`items`|Array|An array of objects representing the commits of the specified repository branch. Each commit object has the following properties:|
+|||`id` (String): Commit id. Will be a hash for Git repositories and a revision number for Subversion repositories.|
+|||`author` (Object): An object with `name` (String) and `email` (String) properties. For Subversion repositories, `name` and `email` will have the same value.|
+|||`date` (String): The commit date and time in RFC 3339 format.|
+|||`message` (String): The commit message.|
+|||`relatedStories` (Array): An array of related user story IDs, if available.|
+|||`mendixVersion` (String, optional): The Mendix version used to make this commit, if available.|
+|`cursors`|Object|An object containing cursors that can be used for pagination:|
+|||`first`: A cursor that can be used to retrieve the first page.|
+|||`prev`: A cursor that can be used to retrieve the previous page. The absence of this property indicates that this is the first page.|
+|||`next`: A cursor that can be used to retrieve the next page. The absence of this property indicates that this is the last page.|
+|||`last`: A cursor that can be used to retrieve the last page.|
+
+##### 4.4.3.1 Example
 
-##### 3.3.2.1 Error Codes
+```json
+{
+  "items": [
+    {
+      "id": "42",
+      "author": {
+        "name": "John Doe",
+        "email": "john.doe@example.com"
+      },
+      "date": "2021-05-31T15:00:00.000Z",
+      "message": "My commit message",
+      "mendixVersion": "8.18.5.18651",
+      "relatedStories": [{ "id": "1234567" }, { "id": "2345678" }]
+    },
+    <more items>...
+  ],
+  "cursors": {
+    "first": "Rmlyc3RQYWdlQ3Vyc29y",
+    "prev": "UHJldmlvdXNQYWdlQ3Vyc29y",
+    "next": "TmV4dFBhZ2VDdXJzb3I=",
+    "last": "TGFzdFBhZ2VDdXJzb3I="
+  }
+}
+```
+##### 4.4.3.2 Error Codes
 
-| HTTP Status | Error code | Description |
+| HTTP Status | Title | Detail |
 | --- | --- | --- |
-| 400 | INVALID_APPID | Invalid AppId. |
-| 404 | APP_NOT_FOUND | App not found. |
-| 404 | BRANCH_NOT_FOUND | There is no branch with name 'branch name'. |
+| 400 | Bad Request | Invalid project ID or branch name |
+| 401 | Unauthorized | Invalid token |
+| 403 | Forbidden | Access denied |
+| 404 | Not Found | Repository or branch not found |
+| 500 | Internal Server Error | Something went wrong |
+
+Error Response format and examples are given in section: [Error response format](#3-error-response-format)
 
-##### 3.3.2.2 Example
+## 5 Webhook for new commits
```

Comment:
Webhooks is not part of TS:Git GA, we shouldn't publish this section.

Reply:
Removed